projects/ion/src/lib/notification/service/notification.service.ts

```diff
@@ -0,0 +1,202 @@
+  }
+
+  public warning(
+    title: string,
+    message: string,
+    options?: NotificationConfigOptions
+  ): void {
+    if (!this.notificationContainerComponentRef)
+      this.createNotificationContainer();
+
+    const notification = this.createNotificationInstance();
+
+    this.configNotification(
+      notification.instance,
+      title,
+      message,
+      options,
+      NOTIFICATION_TYPES.warning
+    );
+
+    notification.hostView.detectChanges();
+    notification.changeDetectorRef.detectChanges();
+
+    this.notificationContainerComponentRef.instance.addNotification(
+      notification
+    );
+
+    this.notificationContainerComponentRef.changeDetectorRef.detectChanges();
+
+    return this.componentSubscriber.next();
+  }
+
+  public error(
+    title: string,
+    message: string,
+    options?: NotificationConfigOptions
+  ): void {
+    if (!this.notificationContainerComponentRef)
+      this.createNotificationContainer();
+
+    const notification = this.createNotificationInstance();
+
+    this.configNotification(
+      notification.instance,
+      title,
+      message,
+      options,
+      NOTIFICATION_TYPES.negative
+    );
+
+    notification.hostView.detectChanges();
+    notification.changeDetectorRef.detectChanges();
+
+    this.notificationContainerComponentRef.instance.addNotification(
+      notification
+    );
+
+    this.notificationContainerComponentRef.changeDetectorRef.detectChanges();
+
+    return this.componentSubscriber.next();
+  }
+
+  private createComponentView(
+    viewRef: ComponentRef<IonNotificationContainerComponent>
+  ): void {
+    this.appRef.attachView(viewRef.hostView);
+    viewRef.changeDetectorRef.detectChanges();
+
+    const notificationElement = viewRef.location.nativeElement;
+    this.document.body.appendChild(notificationElement);
+
+    this.componentSubscriber = new Subject<SafeAny>();
+    this.componentSubscriber.asObservable();
+  }
+
+  private createNotificationContainer(): void {
+    const containerRef = this.componentFactoryResolver
+      .resolveComponentFactory(IonNotificationContainerComponent)
+      .create(this.injector);
+
+    this.notificationContainerComponentRef = containerRef;
+
+    this.createComponentView(this.notificationContainerComponentRef);
+  }
+
+  private createNotificationInstance(): ComponentRef<IonNotificationComponent> {
+    return this.componentFactoryResolver
+      .resolveComponentFactory(IonNotificationComponent)
+      .create(this.injector);
+  }
+
+  private configNotification(
+    notification: IonNotificationComponent,
+    title: string,
+    message: string,
+    options: NotificationConfigOptions,
+    type: StatusType = 'success'
+  ): void {
+    notification.title = title;
+    notification.message = message;
+    notification.type = type;
+
+    if (options) {
+      Object.keys(options).forEach((prop) => {
+        notification[prop] = options[prop];
+      });
+    }
```

Comment:
```suggestion
    if (options) {
      Object.keys(options).forEach((prop) => {
        notification[prop] = options[prop];
      });
    }
```

Do you this to copy object? Case yes, try this.

```
if (options) {
  notification = { ...notification, ...options };
}
```